src/adagrams.js

```diff
@@ -1,6 +1,92 @@
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    const letterQuan = {A: 9, B: 2, C: 2, D: 4, E: 12, F: 2, G: 3, H: 2, I: 9, J: 1, K: 1, L: 4, M: 2, N: 6, O: 8, P: 2, Q: 1, R: 6, S: 4, T: 6, U: 4, V: 2, W: 2, X: 1, Y: 2, Z: 1};
+    
+    // create a pool of letters to draw from
+    let letterPool = [];
+    for (let letter in letterQuan) {
+      for (let i = 0; i < letterQuan[letter]; i++) {
+        letterPool.push(letter);
+      };
+    };
+
+    // shuffle letter pool, source: https://medium.com/@nitinpatel_20236/how-to-shuffle-correctly-shuffle-an-array-in-javascript-15ea3f84bfb
+    // j is a random integer 0 ≤ j ≤ i
+    for (let i = letterPool.length-1; i > 0; i--) {
+      let j = Math.floor(Math.random() * i); 
+      let temp = letterPool[i];
```

Comment:
Both these variables are assigned and never updated so it'd be more appropriate to assign them as `const`.